src/main/java/codeu/controller/NotificationServlet.java

```diff
@@ -2,6 +2,8 @@
 
 import codeu.model.data.User;
 import codeu.model.store.basic.UserStore;
+import codeu.model.data.Event;
```

Comment:
I don’t think you need most of these changes. Only the initialization of the new field in User.java.